<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarMonth</class>
 <widget class="QWidget" name="CalendarMonth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalendarMonth</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="MainWindow" name="YearView">
    <widget class="QPushButton" name="DayViewButton_10">
     <property name="geometry">
      <rect>
       <x>766</x>
       <y>15</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Day</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MonthViewButton_10">
     <property name="geometry">
      <rect>
       <x>642</x>
       <y>15</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Month</string>
     </property>
    </widget>
    <widget class="QPushButton" name="WeekViewButton_10">
     <property name="geometry">
      <rect>
       <x>702</x>
       <y>15</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Week</string>
     </property>
    </widget>
    <widget class="QPushButton" name="YearViewButton_10">
     <property name="geometry">
      <rect>
       <x>592</x>
       <y>15</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
    <zorder>DayViewButton_10</zorder>
    <zorder>WeekViewButton_10</zorder>
    <zorder>MonthViewButton_10</zorder>
    <zorder>YearViewButton_10</zorder>
   </widget>
   <widget class="QWidget" name="MonthView">
    <widget class="QPushButton" name="WeekViewButton_3">
     <property name="geometry">
      <rect>
       <x>702</x>
       <y>15</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Week</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MonthViewButton_3">
     <property name="geometry">
      <rect>
       <x>642</x>
       <y>15</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Month</string>
     </property>
    </widget>
    <widget class="QPushButton" name="YearViewButton_3">
     <property name="geometry">
      <rect>
       <x>592</x>
       <y>15</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DayViewButton_3">
     <property name="geometry">
      <rect>
       <x>766</x>
       <y>15</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Day</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="MonthStack">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>59</y>
       <width>841</width>
       <height>461</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="August" name="August"/>
     <widget class="September" name="September"/>
     <widget class="October" name="October">
      <zorder>horizontalLayoutWidget_3</zorder>
      <zorder>horizontalLayoutWidget_3</zorder>
     </widget>
     <widget class="November" name="November"/>
     <widget class="December" name="December"/>
     <widget class="January" name="January"/>
     <widget class="February" name="February"/>
     <widget class="March" name="March"/>
     <widget class="April" name="April"/>
     <widget class="May" name="May"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>841</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>SUN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string>MON</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TUE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>WED</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>THU</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>FRI</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>SAT</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>DayViewButton_3</zorder>
    <zorder>WeekViewButton_3</zorder>
    <zorder>MonthViewButton_3</zorder>
    <zorder>YearViewButton_3</zorder>
    <zorder>MonthStack</zorder>
    <zorder>horizontalLayoutWidget_3</zorder>
   </widget>
   <widget class="QWidget" name="WeekView">
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>61</y>
       <width>841</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>SUN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string>MON</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TUE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>WED</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>THU</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>FRI</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>SAT</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="FridayCornerLabel">
     <property name="geometry">
      <rect>
       <x>591</x>
       <y>106</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="TuesdayCornerLabel">
     <property name="geometry">
      <rect>
       <x>236</x>
       <y>106</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="WednesdayCornerLabel">
     <property name="geometry">
      <rect>
       <x>354</x>
       <y>106</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="NextButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>490</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLabel" name="MondayCornerLabel">
     <property name="geometry">
      <rect>
       <x>117</x>
       <y>106</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="PrevButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>490</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
    <widget class="QLabel" name="ThursdayCornerLabel">
     <property name="geometry">
      <rect>
       <x>473</x>
       <y>106</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="SundayCornerLabel">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>106</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="SaturdayCornerLabel">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>106</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>841</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="SunPush">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MonPush">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TuePush">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="WedPush">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ThuPush">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FriPush">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SatPush">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgba(255, 255, 255, 0);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>horizontalLayoutWidget_2</zorder>
    <zorder>NextButton</zorder>
    <zorder>PrevButton</zorder>
    <zorder>horizontalLayoutWidget</zorder>
    <zorder>FridayCornerLabel</zorder>
    <zorder>MondayCornerLabel</zorder>
    <zorder>SaturdayCornerLabel</zorder>
    <zorder>SundayCornerLabel</zorder>
    <zorder>ThursdayCornerLabel</zorder>
    <zorder>TuesdayCornerLabel</zorder>
    <zorder>WednesdayCornerLabel</zorder>
   </widget>
   <widget class="calendarday" name="DayView">
    <widget class="QPushButton" name="YearViewButton_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MonthViewButton_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Month</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DayViewButton_2">
     <property name="geometry">
      <rect>
       <x>464</x>
       <y>10</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Day</string>
     </property>
    </widget>
    <widget class="QPushButton" name="WeekViewButton_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Week</string>
     </property>
    </widget>
    <zorder>DayViewButton_2</zorder>
    <zorder>WeekViewButton_2</zorder>
    <zorder>MonthViewButton_2</zorder>
    <zorder>YearViewButton_2</zorder>
   </widget>
   <widget class="QWidget" name="AugustNull"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>calendarday</class>
   <extends>QWidget</extends>
   <header>calendarday.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>August</class>
   <extends>QWidget</extends>
   <header>august.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>September</class>
   <extends>QWidget</extends>
   <header>september.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>October</class>
   <extends>QWidget</extends>
   <header>october.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>November</class>
   <extends>QWidget</extends>
   <header>november.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>December</class>
   <extends>QWidget</extends>
   <header>december.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>January</class>
   <extends>QWidget</extends>
   <header>january.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>February</class>
   <extends>QWidget</extends>
   <header>february.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>March</class>
   <extends>QWidget</extends>
   <header>march.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>April</class>
   <extends>QWidget</extends>
   <header>april.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>May</class>
   <extends>QWidget</extends>
   <header>may.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainWindow</class>
   <extends>QWidget</extends>
   <header>mainwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
